packages/mermaid/src/diagrams/railroad/railroadDiagram.ts

```diff
@@ -0,0 +1,15 @@
+import { DiagramDefinition } from '../../diagram-api/types.js';
+// @ts-ignore: jison doesn't export types
+import parser from './railroadParser.jison';
+import { db } from './railroadDB.js';
+import renderer from './railroadRenderer.js';
+// import { prepareTextForParsing } from './railroadUtils.js';
+
+// const originalParse = parser.parse.bind(parser);
+// parser.parse = (text: string) => originalParse(prepareTextForParsing(text));
```

Comment:
I left this lines here intentionally. I don't know how to implement this in a beautiful way, but there is a need to pre-process diagram code before parsing. Even if we dont need it in the majority of the cases we have to provide developers with a possibility to do it in a standard way before parsing, otherwise everyone will do it his own way